assignment.rb

```diff
@@ -0,0 +1,22 @@
+print "Welcome to the Star Wars Character test\n Please Enter your name: "
+name = gets.chomp
+    puts "Hello #{name}\n"
```

Comment:
In Ruby, tabbing/spaces are used to imply that something is "nested" within something else. However, I don't think that's the case here.